config/kernelci.conf

```diff
@@ -19,6 +19,7 @@ ssh_host: 172.17.0.1
 output: /home/kernelci/output
 plan: check-describe
 lab_config: shell
+target: shell_python
```

Comment:
We already have them in the `pipeline.YAML`. Here we are just settings the default values. Why should we drop them?

Reply:
This isn't actually used. It's OK, we can drop it later when the pipeline YAML starts getting used properly rather than having things hard-coded in `runner.py`.